src/modules/auth/containers/LoginCard.js

```diff
@@ -268,10 +268,10 @@ function LoginCard(props) {
 		setSharedSecretKey(null);
 		setLoginButtonHidden(false);
 		reset({
-			ident: ""
+			username: ""
 		});
 
-		document.getElementById("ident").focus();
+		document.getElementById("username").focus();
```

Comment:
@petrKavulok Pls check if `focus` works properly (it should focus to username input field automatically)

Reply:
it does :-)